tests/test_grid3d/test_grid.py

```diff
@@ -666,20 +664,16 @@ def test_bulkvol():
     logger.info("Sum this: %s", bulk.values.sum())
     logger.info("Sum RMS: %s", cellvol_rms.values.sum())
 
-    assert bulk.values.sum() == pytest.approx(cellvol_rms.values.sum(), rel=0.001)
 
+@pytest.mark.benchmark(group="bulkvol")
```

Comment:
I have not used pytest-benchmark before.
When does a test such as this fail?

Reply:
It doesn't fail but it provides the benchmark statistics on completion. That at least means we don't remove any tests that had been added at some point, but once this code changes we will have to create some more unit tests that asserts the behavior.